router/pods_test.go

```diff
@@ -800,30 +800,26 @@ func TestGetHealthCheckFromPodPort(t *testing.T) {
 
 // TestValidatePath tests the internal function to validate a proper path used in annotations
 func TestValidatePath(t *testing.T) {
-	testNoPrefix := "test"
-	testPathFail := "/test/%2a/%"
-	testPathPass := "/test/%2a/aa/a"
-	testPathPassProperEncoding := "/test/par%2ate/aa/a"
-	testPathFailInvalidEncoding := "/test/hello%zzworld/aa/a"
```

Comment:
So that was actually something Martin brought up when i first talk to him about it. He proposed we change `path` to something more descriptive like `pathPattern` or something i forgot what he proposed.

Though  i don't think two separate values is very useful, it's still the same thing your providing just in one case your providing the simplest proper URI format.